### ISSUE TITLE
Kleur contrast

### ISSUE THREAD

**Issue by misspastelwitch** [misspastelwitch/the-client-website#8]
Kleurcontrast meets WCAG AA guidelines, but fails AAA guidelines for 1.4.6 - The visual presentation of text and images of text has a contrast ratio of at least 7:1. The current ratio is 6.7:1. I will fix this by changing the text colour from  #99004d to #6b0237, which turns the contrast ratio to 9.7:1

![Screenshot 2024-10-21 110810](https://github.com/user-attachments/assets/06513a7d-182d-4cef-a93a-78ab7bf3d41a)
Text colour before

![Screenshot 2024-10-21 111124](https://github.com/user-attachments/assets/866b1036-058c-4c24-a3b5-7d02ecd42829)
Text colour with improved contrast
